### PR TITLE
chore(weave): Adds Tracer CTAs and cleans up related notebooks

### PR DESCRIPTION
**Pull request by tssweeney**
Screenshot 2023-09-19 at 16 44 44

### CODE REVIEW

File: weave/monitoring/langchain.py [hunk 1]
```diff
@@ -79,6 +79,29 @@ def _hash_id(s: str) -> str:
     return hashlib.md5(s.encode("utf-8")).hexdigest()[:16]
 
 
+patched = False
+
+
+# This should be committed to the langchain repo
```
Comment (staceysv) [inline weave/monitoring/langchain.py:85]: NICE!!!

File: weave/panels_py/instructions/panel_trace_monitor.md [hunk 2]
```diff
@@ -2,6 +2,66 @@
...
+- (Easy) Via 3rd party integrations (eg. `Langchain`)
 - (Medium) Via decorating existing code with `@trace`
-- (Advanced) Via low-level construction of Spans.
+- (Advanced) Via low-level construction of Spans using `TraceSpanDict`.
+
+> Already using [W&B Tracer](https://docs.wandb.ai/guides/prompts)? Stay tuned: you soon will be able to load these traces in this dashboard!
+
+### Via Langchain Tracer
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/wandb/weave/blob/tim%2Fmake_tracer_a_featured_template/examples/prompts/trace_debugging/trace_quickstart_langchain.ipynb)
```
Comment (staceysv) [inline weave/panels_py/instructions/panel_trace_monitor.md:13]: minor todo? update these urls relative to main once merged?

File: weave/panels_py/instructions/panel_trace_monitor.md [hunk 3]
```diff
@@ -2,6 +2,66 @@
...
+
+If you are already using `Langchain`, simply create a tracer and add it to your next call:
```
Comment (staceysv) [inline weave/panels_py/instructions/panel_trace_monitor.md:15]: minor: explicitly saying this is a callback would help imho, e.g. "create a tracer and add it as a callback" or "simply create a tracer callback"?

File: weave/panels_py/instructions/panel_trace_monitor.md (hunk 2 above)
Comment (tssweeney) [reply weave/panels_py/instructions/panel_trace_monitor.md:13]: yup! exactly

File: weave/panels_py/instructions/panel_trace_monitor.md (hunk 3 above)
Comment (tssweeney) [reply weave/panels_py/instructions/panel_trace_monitor.md:15]: adding

**Review by staceysv — APPROVED** [review]
this is awesome—excited to get these in & we can keep polishing example details!

**Pull request merged**